api/server/volume.go

```diff
@@ -598,5 +697,9 @@ func (vd *volAPI) Routes() []*Route {
 		{verb: "POST", path: snapPath("", volume.APIVersion), fn: vd.snap},
 		{verb: "GET", path: snapPath("", volume.APIVersion), fn: vd.snapEnumerate},
 		{verb: "POST", path: snapPath("/restore/{id}", volume.APIVersion), fn: vd.restore},
+		{verb: "GET", path: credsPath("/credslist", volume.APIVersion), fn: vd.credsList},
+		{verb: "POST", path: credsPath("/credscreate", volume.APIVersion), fn: vd.credsCreate},
+		{verb: "DELETE", path: credsPath("/credsdelete", volume.APIVersion), fn: vd.credsDelete},
+		{verb: "POST", path: credsPath("/credsvalidate", volume.APIVersion), fn: vd.credsValidate},
```

Comment:
not sure what this API is doing ?

Reply:
Returns no error if it can access cloud provider with the given credential.